pallets/xc-asset-config/src/lib.rs

```diff
@@ -139,6 +139,7 @@ pub mod pallet {
         /// a AssetLocation
         type AssetId: Member + Parameter + Default + Copy + HasCompact + MaxEncodedLen;
 
+        /// TODO: this has essentially been deprecated and can be removed.
         /// Callback handling for cross-chain asset registration or unregistration.
         type XcAssetChanged: XcAssetChanged<Self>;
```

Comment:
Why not just remove it now? It's not like anyone is using this pallet besides us

Reply:
I left it so it would get commented - I don't see the use for it, but maybe someone else does. I guess not 🙂